test/Microsoft.AspNet.FileProviders.Embedded.Tests/project.json

```diff
@@ -2,17 +2,23 @@
     "dependencies": {
         "Microsoft.AspNet.FileProviders.Embedded": "1.0.0-*",
         "Microsoft.AspNet.Testing": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
+        "xunit": "2.1.0"
     },
+    "testRunner": "xunit",
     "commands": {
         "test": "xunit.runner.aspnet"
```

Comment:
I'm not super familiar with how the testRunner works, but is this property still used \ needed? I was under the impression `commands` doesn't work in dotnet.